biohansel/create/io/output.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+import pandas as pd
+
+from typing import List
+
+from biohansel.create.schema_generator import get_sequence_string
+
+def write_sequence_file(output_directory: str, df_list: List[pd.DataFrame],
```

Comment:
Please extract a function that returns a generator that yields the tile sequences as a string in FASTA format.

Reply:
yes this was addressed in b3479441f774fa2b3fc3f01a9a8e74656a4aaaeb